src/style.css

```diff
@@ -1 +1,68 @@
 /* Styles go here */
+body {
+  margin: 0;
+  font-family: "Roboto", sans-serif;
+  min-width: 1024px;
+  max-width: 1200px;
+}
+
+.header {
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+  padding: 0 50px;
+}
+
+.nav_list {
+  margin: 0;
+  list-style: none;
+  display: flex;
+  padding: 0;
+
+}
+
+.nav_item {
+  margin-left: 20px;
+}
+
+.nav_item:first-child {
```

Comment:
It's more efficient to use :not(:first-child) selector instead of :first-child to remove margin-left on the first .nav_item.

Reply:
done